src/js/common.js

```diff
@@ -1,12 +1,7 @@
-function trace(text) {
-  // This function is used for logging.
-  if (text[text.length - 1] === '\n') {
-    text = text.substring(0, text.length - 1);
-  }
-  if (window.performance) {
-    var now = (window.performance.now() / 1000).toFixed(3);
```

Comment:
`performance.now` is supported by Chrome 25+ on Android; Chrome 20+, IE10+ and Firefox 15+ on desktop, so I don't think we need worry.